src/concrete/ml/common/serialization/__init__.py

```diff
@@ -12,7 +12,6 @@
     activation.CELU,
     activation.ELU,
     activation.GELU,
-    activation.Hardshrink,
```

Comment:
why is that ? if me remove it shouldn't it be removed from tests and doc support as well ? also, worth a comment somewhere saying why we do not support this activation

Reply:
hardshrink has export warnings due to an old implementation of it in pytorch. there are very few use cases for hardshrink anyway. I removed it from the other lists

Reply:
got it ok, worth that you update https://github.com/zama-ai/concrete-ml-internal/issues/335 to tell why it's not supported anymore then !